src/test/java/io/supertokens/test/multitenant/ConfigTest.java

```diff
@@ -358,6 +366,538 @@ public void testDifferentWaysToGetConfigBasedOnConnectionURIAndTenantId()
                 (long) 144004);
 
 
+        process.kill();
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STOPPED));
+    }
+
+    @Test
+    public void testMappingSameUserPoolToDifferentConnectionURIThrowsError()
+            throws InterruptedException, IOException, TenantOrAppNotFoundException {
+        String[] args = {"../"};
+
```

Comment:
add a new test that will randomly generate a tenant:
- for each tenant config's value, you can specify a generateValue function, and expected outcome -> 
   - thirdpartId 
      - generate function will return a random string (UUID) -> expected outcome is success (UUID)
      -  generate function will return a null  -> expected outcome is InvalidConfigException
   - infomap
      - generate function will return null -> outcome is empty map

- You can use java reflection -> enforce that each config that we have is tested.